### PR TITLE
30 min  package refresh

### PR DESCRIPTION
**Pull request by aballerr**
As far as I can tell, the gui only checks the update status on packages when the gui is initialized.  So you have to reset the gui in order to see the needed updates.  I can reduce the timeframe, but doing this at least once a day would make sense to me

### CODE REVIEW

File: svelte/src/libs/stores/pkgs.ts [hunk 1]
```diff
@@ -166,6 +166,10 @@ const init = async function () {
     await refreshPackages();
     await monitorTeaDir();
     initialized = true;
+
+    setInterval(async () => {
+      await refreshPackages();
+    }, 1000 * 60 * 30);
```
Comment (getneil) [inline svelte/src/libs/stores/pkgs.ts:172]: in line 247 there is a refresh, did you figure out why it was not working?

Comment (aballerr) [reply svelte/src/libs/stores/pkgs.ts:172]: it looks like it is only a timeout, so maybe that's why?

Comment (aballerr) [reply svelte/src/libs/stores/pkgs.ts:172]: I didn't notice that, I'm not really sure.  Maybe it's best if someone else can verify they have the same issue first.  For me, I just notice that if I don't restart my tea gui then I don't see new updates

Comment (aballerr) [reply svelte/src/libs/stores/pkgs.ts:172]: I'm not really familiar with svelete.  Is it possible ondestroy is called which would prevent the fresh from actually refreshing packages?

Comment (getneil) [reply svelte/src/libs/stores/pkgs.ts:172]: ill dig deeper into this today but @ABevier  implemeted the refresh strategy here.

**Pull request closed without merging**